index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
```

Comment:
- [ ] Nice job adding a logo to your page 👍. Please consider wrapping the logo image in an `a` tag as the logo should be a link that leads to the homepage. See my suggestion below.

```suggestion
                <a href="#" class="logo">
                     <img src="images/MyLogo.png" alt="Logo">
                </a>
```